samples_project/Assets/SampleViewer/Samples/StreamLayer/readme.md

```diff
@@ -1,30 +1,52 @@
-# Connect to a Stream Layer
+# Subscribe to a stream Service
```

Comment:
I think the current title is still valid and OK. But I feel like this is better after reading the old blog article...
https://community.esri.com/t5/arcgis-geoevent-server-blog/howto-subscribe-to-a-geoevent-server-stream/ba-p/889351

Reply:
I dont like `Service` being capitalized after `stream` is lowercase. Feels wrong.

Reply:
Good point. I'm gonna change it to `stream service`

Reply:
```suggestion
# Subscribe to a stream service
```